test/v1/connectors/DeleteAnnualSubmissionConnectorSpec.scala

```diff
@@ -28,49 +25,53 @@ import scala.concurrent.Future
 
 class DeleteAnnualSubmissionConnectorSpec extends ConnectorSpec {
 
-  val taxYear: String           = "2017-18"
-  val downstreamTaxYear: String = "2018"
-  val nino: String              = "AA123456A"
-  val businessId: String        = "XAIS12345678910"
+  val nino: String       = "AA123456A"
+  val businessId: String = "XAIS12345678910"
 
-  val request: DeleteAnnualSubmissionRequest = DeleteAnnualSubmissionRequest(
-    nino = Nino(nino),
-    businessId = BusinessId(businessId),
-    taxYear = TaxYear.fromMtd(taxYear)
-  )
+  trait Test {
+    _: ConnectorTest =>
+    def taxYear: TaxYear
 
-  class Test extends MockHttpClient with MockAppConfig {
-
-    val connector: DeleteAnnualSubmissionConnector = new DeleteAnnualSubmissionConnector(
+    protected val connector: DeleteAnnualSubmissionConnector = new DeleteAnnualSubmissionConnector(
       http = mockHttpClient,
       appConfig = mockAppConfig
     )
 
-    MockAppConfig.ifsBaseUrl returns baseUrl
-    MockAppConfig.ifsToken returns "ifs-token"
-    MockAppConfig.ifsEnvironment returns "ifs-environment"
-    MockAppConfig.ifsEnvironmentHeaders returns Some(allowedIfsHeaders)
+    protected val request: DeleteAnnualSubmissionRequest = DeleteAnnualSubmissionRequest(
+      nino = Nino(nino),
+      businessId = BusinessId(businessId),
+      taxYear = taxYear
+    )
+
   }
 
   "deleteAnnualSubmission" should {
     "return a 204 with no body" when {
-      "the downstream call is successful" in new Test {
+      "the downstream call is successful when not taxYearSpecific" in new DesTest with Test {
+        def taxYear: TaxYear = TaxYear.fromMtd("2017-18")
+
+        val outcome = Right(ResponseWrapper(correlationId, ()))
+
+        willPut(
```

Comment:
Can we added a negative test(s) for when errors are returned? See [here](https://github.com/hmrc/property-business-api/blob/main/test/v2/connectors/AmendUkPropertyPeriodSummaryConnectorSpec.scala)

Reply:
Done